### ISSUE TITLE
Profile page crash: "Param 0 must have a .id of type "feedId"

### ISSUE THREAD

**Issue by cinnamon-bun** [fraction/oasis#26]
**What version of this package are you using?**
2.9.4

**What operating system, Node.js, and npm version?**
OS X 10.11.6
node v10.8.0
npm 6.3.0

**What happened?**
The Profile page shows an error message instead of any content.

This doesn't happen on my `/author/xxxxxx` page, only on `/profile`

This is the error when Oasis is using Patchwork's ssb-server:
```
Error: non-error thrown: {"message":"Param 0 must have a .id of type \"feedId\"","name":"UsageError","stack":"UsageError: Param 0 must have a .id of type \"feedId\"\n    at new ZE (/Applications/Patchwork-3.17.1.app/Contents/Resources/app.asar/node_modules/zerr/index.js:11:42)\n    at ZError (/Applications/Patchwork-3.17.1.app/Contents/Resources/app.asar/node_modules/zerr/index.js:13:14)\n    at Object.createUserStreamOpts (/Applications/Patchwork-3.17.1.app/Contents/Resources/app.asar/node_modules/ssb-db/lib/validators.js:146:14)\n    at validate (/Applications/Patchwork-3.17.1.app/Contents/Resources/app.asar/node_modules/muxrpc-validation/index.js:153:15)\n    at Object.<anonymous> (/Applications/Patchwork-3.17.1.app/Contents/Resources/app.asar/node_modules/muxrpc-validation/index.js:95:17)\n    at Object.hooked (/Applications/Patchwork-3.17.1.app/Contents/Resources/app.asar/node_modules/hoox/index.js:10:15)\n    at Object.localCall (/Applications/Patchwork-3.17.1.app/Contents/Resources/app.asar/node_modules/muxrpc/local-api.js:31:29)\n    at Object.<anonymous> (/Applications/Patchwork-3.17.1.app/Contents/Resources/app.asar/node_modules/muxrpc/local-api.js:37:22)\n    at PacketStreamSubstream.stream.read (/Applications/Patchwork-3.17.1.app/Contents/Resources/app.asar/node_modules/muxrpc/stream.js:69:23)\n    at PacketStream._onstream (/Applications/Patchwork-3.17.1.app/Contents/Resources/app.asar/node_modules/packet-stream/index.js:228:11)"}
    at Object.onerror (/Users/xxx/projects/ssb/oasis/node_modules/koa/lib/context.js:113:40)
    at onerror (/Users/xxx/projects/ssb/oasis/node_modules/koa/lib/application.js:148:32)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

And when Oasis is running its own ssb-server:
```
Error: non-error thrown: {"message":"Param 0 must have a .id of type \"feedId\"","name":"UsageError","stack":"UsageError: Param 0 must have a .id of type \"feedId\"\n    at new ZE (/Users/xxx/projects/ssb/oasis/node_modules/zerr/index.js:11:42)\n    at ZError (/Users/xxx/projects/ssb/oasis/node_modules/zerr/index.js:13:14)\n    at Object.createUserStreamOpts (/Users/xxx/projects/ssb/oasis/node_modules/ssb-db/lib/validators.js:146:14)\n    at validate (/Users/xxx/projects/ssb/oasis/node_modules/muxrpc-validation/index.js:153:15)\n    at Object.<anonymous> (/Users/xxx/projects/ssb/oasis/node_modules/muxrpc-validation/index.js:95:17)\n    at Object.hooked (/Users/xxx/projects/ssb/oasis/node_modules/hoox/index.js:10:15)\n    at Object.localCall (/Users/xxx/projects/ssb/oasis/node_modules/secret-stack/node_modules/muxrpc/local-api.js:31:29)\n    at Object.<anonymous> (/Users/xxx/projects/ssb/oasis/node_modules/secret-stack/node_modules/muxrpc/local-api.js:37:22)\n    at PacketStreamSubstream.stream.read (/Users/xxx/projects/ssb/oasis/node_modules/secret-stack/node_modules/muxrpc/stream.js:69:23)\n    at PacketStream._onstream (/Users/xxx/projects/ssb/oasis/node_modules/packet-stream/index.js:228:11)"}
    at Object.onerror (/Users/xxx/projects/ssb/oasis/node_modules/koa/lib/context.js:113:40)
    at onerror (/Users/xxx/projects/ssb/oasis/node_modules/koa/lib/application.js:148:32)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Are you willing to submit a pull request to fix this bug?**
No, I don't know how to fix it.

**Comment by christianbundy** [comment]
Thanks for the bug report! I'm hopping off for the night but I'll plan to take a look at this tomorrow.

**Comment by christianbundy** [comment]
I'm wondering whether maybe you might have an older version of ssb-client, could you try running `npm install` or `yarn install` to see if that fixes it? Sorry these error message are so useless.

**Comment by cinnamon-bun** [comment]
Fixed in the same way as #29 - ssb-client was out of date.  Works now  🌟
